projects/packages/codesniffer/tests/php/tests/files/i18n.php.report

```diff
@@ -22,4 +22,5 @@
  64 | ERROR | [x] Mismatched text domain. Expected 'domain' but got 'wrongdomain'. (Jetpack.Functions.I18n.TextDomainMismatch)
  65 | ERROR | [x] Mismatched text domain. Expected 'domain' but got 'wrongdomain'. (Jetpack.Functions.I18n.TextDomainMismatch)
  72 | ERROR | [x] Mismatched text domain. Expected 'domain' but got 'wrongdomain'. (Jetpack.Functions.I18n.TextDomainMismatch)
- 77 | ERROR | [ ] The domain must not contain interpolated variables. Found $domain. (Jetpack.Functions.I18n.InterpolatedVariable)
+ 77 | ERROR | [ ] The domain must not contain interpolated variables. Found $domain.
+    |       |     (Jetpack.Functions.I18n.InterpolatedVariable)
```

Comment:
I had started on a PR to update codesniffer myself, but I got sidetracked by seeing this unnecessary change and figuring out the details to file https://github.com/squizlabs/PHP_CodeSniffer/issues/3924 😀